tests/php/RestApi/Controllers/ProductAttributes.php

```diff
@@ -104,10 +104,10 @@ public function test_get_attribute_contributor() {
 		wp_set_current_user( $this->contributor );
 		$request = new WP_REST_Request( 'GET', $this->endpoint . '/products/attributes/' . $this->attr_size['attribute_id'] );
 
-		$response  = $this->server->dispatch( $request );
-		$attribute = $response->get_data();
+		$response = $this->server->dispatch( $request );
+		$data     = json_decode( wp_json_encode( $response->get_data() ), true ); // Converts objects to arrays.
 		$this->assertEquals( 200, $response->get_status() );
-		$this->assertEquals( $this->attr_size['attribute_id'], $attribute['id'] );
-		$this->assertEquals( $this->attr_size['attribute_name'], $attribute['name'] );
+		$this->assertEquals( $this->attr_size['attribute_id'], $data['id'] );
+		$this->assertEquals( $this->attr_size['attribute_name'], $data['name'] );
```

Comment:
This was added as a result of https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/1553#discussion_r367067713 - rather than deal with objects and rewrite existing tests I just wanted to remain dealing with assoc arrays.